Adjustments to the new Cargo sprites, and updates to a few missed sprites

OrionTheFox opened the pull request:
## About The Pull Request
There were a few things about that last cargo resprite that people didn't quite agree with. The resprite had heart, but so did the complaints. This aims to be the best of both worlds, keeping the "dark cargo" resprite but making it blend better with the rest of the game.

First, this gets a few sprites that were missed: 
Winter Coat
QM Winter Coat
QM Cloak
Beret (This is a Greyscale item so icon quality isn't something I can change. Its just the best color I could get it now)

This also adds a skirt variant of the cargo shorts, the cargo shortskirt

As for the old sprites:
Fixes the contrast/palette on the new cargo sprites to make them less violently black and dark
![image](https://github.com/tgstation/tgstation/assets/76465278/a0dff7d1-f33b-4fa4-b7ad-8de79ac312c4)

Redoes the Object sprites to better match the rest of the codebase's objects, as well as fixing the same contrast/palette issue mentioned above
![image](https://github.com/tgstation/tgstation/assets/76465278/f708689b-e3cf-44ba-8ea8-2d7d0ce83173)

And finally, adds **proper** rolled-down states to all of them
![image](https://github.com/tgstation/tgstation/assets/76465278/e765ca9b-b167-4258-8d29-1948e09b8dd5)

Smaller things:
- Renames "outfits" to "uniforms" (I've never seen any clothes called "outfits" before and it seemed very out of place)
- Adjusts the mining suits code to have Name above Description like every other item in this file

No Sealed I'm not doing the banner. And anybody who asks about Cargorilla, they had a huge stockpile of the old uniform in the gorilla's size that nobody else could really wear so they need to use it all up
## Why It's Good For The Game
There was a lot of player (and spriter) feedback that should have been taken into consideration; the one that stood out most to me was @Draganfrukts, who even gave an example of a better palette - I attempted to mesh both his idea and the original colors into a combination that fits the game environment a lot better.
## Changelog
:cl:
add: Cargo now once again has access to a "Cargo Shortskirt", a skirt variant of their shorts!
image: tweaks and adjustments to the Cargo resprite have been made, as well as respriting some missed icons such as the Winter Coats and QM Cloak
/:cl:

Commit:
Recolor Wintercoats to fit new cargo

Commit:
hats recolor

Commit:
uniforms "re-sprite" mostly just recolor but tweaks to make them similar to the rest of the codebase

Commit:
jacket re-color and small tweaks (i hate obj icons)

Commit:
i've never seen uniforms called "outfits" before

Commit:
beret color fix (still bad but thats GAGS for you)

Commit:
add alt skirt to match the cargo shorts

Commit:
QM cloak

OrionTheFox commented:
> Screenshot tests failed!
> 
> [Download zip file of new screenshots.](https://file.house/CTC4.zip)
> ## Diffs
> See snapshot diffs
> Name 	Expected image 	Produced image 	Diff
> screenshot_antag_icons_bloodbrother 	![](https://camo.githubusercontent.com/833354058624b760dec74eef67ddbd82b2d05ba507d10ee3af05e9ffb8f4fad6/68747470733a2f2f66696c652e686f7573652f7a7a754a2e706e67) 	![](https://camo.githubusercontent.com/b0d184b993becd8d7e2a7d96d97d415d4c91a8989264fac27781d8c3f01aa54c/68747470733a2f2f66696c652e686f7573652f6b7253702e706e67) 	![](https://camo.githubusercontent.com/a25f07c22d3bd79934ff91ada45ef0c9f2bdda3a75e5fa681d61a2ec34ff7a9b/68747470733a2f2f66696c652e686f7573652f675f61392e706e67)
> ## Help
> What is this?
> I am changing sprites, it's supposed to look different.
> This is a false positive.

curse you.

Commit:
fix screenshot tests

Serxule commented:
old ones are still better but atleast these fit _more_

EricZilla commented:
So I'm gonna go over them by outfit I think, this is gonna be pretty disorganized. I'm literally some mcnobody who has like 3 merged PRs but I'm also a cargo main so this is important to me. 

I'm not gonna pretend the re-imagining was perfect, I think Viro tried to do too many things at once and it got sloppy. (Sorry dude). I'm also not gonna pretend if I was Viro I wouldn't be really insulted to see someone changing up big aesthetic choices I made for the sake of consistency when TG is specifically meant NOT to have a set style, its okay to push new grounds here.

I had to get that out before getting to the actual meat and potatos, some of these things are direct improvements, like the cargo jacket, just looks better in your version, but some things are corrections in the wrong direction I think, like the QM jacket not being huge and floppy, why would you do this I loved dressing like an unmade bed.

I'm rambling sorry. Going to go by fit.

Fit 1. Cargo Tech.
1. The jacket looks great, it was weirdly flat if you didn't have a backpack.
2. The new jeans, are worse. The asymmetrical jeans had a very cool and lived in vibe that I felt really pushed Blue Collar more than anything else we'd had before. I'm not saying the new pants are BAD, just that I liked Viro's more, your jeans actually look really nice and I would absolutely like to see some of the other jeans in the game reworked to use that style.
3. As for the top, you made the sleeves tighter, which makes sense. I'd say that on its own merits the change. Could take or leave the new shading. Not an improvement not a downgrade just another very good looking sprite.
4. The Skirt, looks fine, I don't think the black having lots of contrast was a real issue, otherwise the southskirt has a really dark shade on the bottom edge that doesnt match the other directions

Fit 2. Cargo Shorts
1. I prefer your shorts to Viro's for oddly the opposite reason, if they are both the same kind of jeans it feels kind of weird, yours look more like canvas, and that makes a small lizard in my head happy.
2. The shirt. You changed the brightest color but didn't change the shading colors, I'd like to see it overall more orange so its less samey, you are going in the right direction, but it still needs something. I don't know what that something is.

Fit 3. All the outerwear!
1. Winter coats! They look great! Almost no notes. The QMs fur lining is a little too close to the brown of the coat now, maybe use the same color as the tech coat to compensate?
2. The fucking cloak! I hated the old one, and would maybe wear the new one! Good shit.
3. The hat looks different but I can barely tell why. Feels... good?

Fit 4. The Quartermaster. Mr. 25% himself
1. Gotta start with the coat. Bring back the rumples! Make it baggy we love the new coat, this man needs to get caught in machinery. You fixed the front being longer, but personally I think making the back and sides longer would have been a better fix. Shading looks good!
2. Just noticed it uses the same jeans as the cargo tech, this is perfect for QM, especially if you go with the asymmetric jeans for tech, now they have similar style, but it looks like techs do more work, thus the scuffs. They were dress pants originally and I didn't like that since QM is like, the only head that actually does shit with their minions.
3. And to end on a high note Viro QM shirt had nipples and I couldn't unsee it. I like yours.

Thats my thoughts on everything, I tried to be as impartial as I could, you definitely did some good work here, even the stuff where I prefer the Viro version, that doesn't mean yours wasn't good, everyone here is talented.

OrionTheFox commented:
> So I'm gonna go over them by outfit I think, this is gonna be pretty disorganized. I'm literally some mcnobody who has like 3 merged PRs but I'm also a cargo main so this is important to me.

Thanks for the in-depth review, if others share the opinions or don't mind adjustments to fit it, I think lots of this has some good merit. Especially the comparison of the CT and QM pants and how the CT ones being uneven was actually a perk

Jezzared commented:
![268e52ab-b23b-4a08-b1c7-eb8844470b86_1920x1080](https://github.com/tgstation/tgstation/assets/96926864/bcb8fd98-57e6-4ab4-a174-1b7ad2ee9e6f)

Jokes aside, I actually really _love_ these. While I was very iffy on the originals, this moves in the same direction that had while helping with the cohesion of the department. Cargo being out of tune color-wise was bugging me greatly so that's a big plus. Additionally, adding the brown to the sleeves helps with visual clarity when wearing security armor vests or other such chest items.

I could wear these. Nay, I want to play techie just to wear these. I think you and Viro should both feel proud for that. Just wish the original had shipped in a slightly more put-together state so I could have felt that way from the beginning.

cocothegogo commented:
im sorry these just look really bad, dark brown and black just isn't a nice colour combination

EricZilla commented:
> im sorry these just look really bad, dark brown and black just isn't a nice colour combination

That ship has sailed my friend, look at the before and afters

RedBaronFlyer commented:
I still absolutely detest the new cargo respriting as a whole, but these are a very big improvement over the currently resprited ones.

I just don’t like that cargo are now not wearing jumpsuits (and seem to be wearing t-shirts and black shorts) and they still look like extras on a generic sci-if show.

It’s also nice that they aren’t just dried mustard brown now. Still miss my golden cargo uniform though.

Sealed101 commented:
> Everything not being resprited (winter coats for instance) in the original PR + uniforms not being able to be adjusted (rolled down) for surgery showed that it wasn’t a finished project. Yet was still merged anyway.

more fuel to the mosin nut

EricZilla commented:
Id like to point out the cargo outfits could take their shirts off for surgery, people just too umbrage that the shirt wasn't wrapped around the waist. With the QM they rolled up their sleeves instead, which was then swapped to doing the invisible shirt instead. They were stylistic choices not unfinish products. Everyone just keeps calling it unfinished for whatever reason.

Sealed101 commented on code/modules/clothing/under/jobs/cargo.dm:
```diff
@@ -53,7 +58,7 @@
 /obj/item/clothing/under/rank/cargo/miner/lavaland
-	desc = "A green uniform for operating in hazardous environments."
 	name = "shaft miner's jumpsuit"
+	desc = "A green uniform for operating in hazardous environments."
```
since you've touched this i'm going to ask to rectify this trolling
```suggestion
	desc = "A grey uniform for operating in hazardous environments."
```
![изображение](https://github.com/tgstation/tgstation/assets/75863639/3424f026-d110-47d8-8a23-892278e9b0bb)

OrionTheFox replied:
What if I told you...
![image](https://github.com/tgstation/tgstation/assets/76465278/f4f672f8-a34a-4692-9620-6c56077d33d2)
its a green grey :trollface:

CoiledLamb commented:
> take their shirts off for surgery, people just too umbrage that the shirt wasn't wrapped around the waist. With the QM they rolled up their sleeves instead, which was then swapped to doing the invisible shirt instead. They were stylistic choices not unfinished products. Everyone just keeps calling it unfinished for whatever reason.

it's because the stylistic choices here end up being the lazy route, as orion proved it's very feasible to just add wrapped states that are inherently more clear, because we already know what those wrapped states mean.

LSommers commented:
Well, it's a start at least.

EricZilla commented:
> it's because the stylistic choices here end up being the lazy route, as orion proved it's very feasible to just add wrapped states that are inherently more clear, because we already know what those wrapped states mean.

This is a style issue not an ethics issue, just say you prefer the realism of seeing the unrolled shirt and move on

potato1234x commented:
original ones were 100% better but these are good

Commit:
As funny as it would be to leave the Miner Suit description as "green" I miiight as well fix it.

Does it matter how we spell Grey? Gray? I've seen both Greyscale and Grayscale in the code and that's bound to cause confusion with a component or something eventually

Co-authored-by: Sealed101 <cool.bullseye@yandex.ru>

Wallemations commented:
Call me a Tesla the way I'm merging in ways people can't stand
